### ISSUE TITLE
Upgrade Django-coverage-plugin once it works with Python 3.7+

### ISSUE THREAD

**Issue by mikkonie** [bihealth/sodar-core#124]
[django-coverage-plugin](https://github.com/nedbat/django_coverage_plugin) displays erratic behaviour with Python 3.7. Noticed this after #121 and #123.

When this started failing I tested a few combinations:
* Workstation + Python 3.7 + django-coverage-plugin==1.6.0 + test.py = FAIL
* Workstation + Python 3.7 + django-coverage-plugin==1.6.0 + test_taskflow.py = OK (??)
* Workstation + Python 3.7 + django-coverage-plugin==1.5.0 + test.py = OK
* Workstation + Python 3.7 + django-coverage-plugin==1.5.0 + test_taskflow.py = OK
* GitLab-CI + Python 3.7 + django-coverage-plugin==1.6.0 + test.py = FAIL
* GitLab-CI + Python 3.7 + django-coverage-plugin==1.5.0 + test.py = FAIL
* GitLab-CI + Python 3.6 + django-coverage-plugin==1.6.0 + test.py = OK

In GitLab-CI, the failures present the following warning:

```
Coverage.py warning: Disabling plug-in 'django_coverage_plugin.DjangoTemplatePlugin' due to an exception:
Traceback (most recent call last):
  File "/builds/CUBI_Engineering/CUBI_Data_Mgmt/sodar_core/env/lib/python3.7/site-packages/django/template/utils.py", line 65, in __getitem__
    return self._engines[alias]
KeyError: 'django'
```

The results are not consistent, but it indeed seems Python 3.7 is an issue. It's not listed as one of the supported Python versions for the plugin either. Alas, it is the recommended and supported Python version for Django 1.11.17+..

Will submit a ticket for the project and postpone upgrading to 3.7 for now.
